cmd/seed-controller-manager/controllers.go

```diff
@@ -31,6 +31,7 @@ import (
 	clustertemplatecontroller "k8c.io/kubermatic/v2/pkg/controller/seed-controller-manager/cluster-template-controller"
 	seedconstraintsynchronizer "k8c.io/kubermatic/v2/pkg/controller/seed-controller-manager/constraint-controller"
 	constrainttemplatecontroller "k8c.io/kubermatic/v2/pkg/controller/seed-controller-manager/constraint-template-controller"
+	encryptioncontroller "k8c.io/kubermatic/v2/pkg/controller/seed-controller-manager/encryption"
```

Comment:
What does this controller encrypt? Can we make its directory clearer maybe? `etcd-encryption`?